src/main/java/com/senlainc/warsaw/tyurin/util/Constants.java

```diff
@@ -0,0 +1,18 @@
+package com.senlainc.warsaw.tyurin.util;
+
+public class Constants {
+    public static String PATH_TO_CRAFTSMEN_CSV = "src\\main\\resources\\craftsmen.csv";
```

Comment:
When you use relative paths, you may experience problems running the application on other operating systems. Your property files are already in resources, which means they are in the classpath. You can specify paths taking into account the classpath, i.e. PATH_TO_PROPERTIES = "application.properties";

Additionally, you just need to change how you read property files, don't refer to them as files, but use the classloader to refer to them as resources